README.md

```diff
@@ -53,7 +53,7 @@ Building Skulpt is straightforward:
 
 1. Clone the repository from GitHub, ideally using your own fork if you're planning on making any contributions
 2. Install node.js
-3. Install the jscs, jshint and jsdoc node modules using `npm install -g jscs jshint jsdoc` (you may need to use `sudo` to run this command)
+3. Install the required dependencies using `npm install`
 4. Navigate to the repository and run `./skulpt.py dist`
```

Comment:
Navigate to the repository and run `./skulpt.py dist` 
-> 
Navigate to the repository and run `npm run build`